db/seeds/posts/tos.html

```diff
@@ -1,30 +1,5 @@
-</ul>
-<h2 id="3-user-generated-content">3. User-Generated Content</h2>
-<p>If you sign up for an account, you will be able to submit questions, answers, comments, and other types of posts, some of which may include images, video, or other media (collectively, &quot;User-Generated Content&quot; or &quot;Content&quot;).</p>
-<p>By posting here, you license any such Content under the terms of one of (in increasing order of precedence) (a) the site default license at the time of creating your post; (b) the license of the category in which you post, or (c) the license you choose from the available list at the time of creating your post &mdash; the &quot;Applicable License&quot;. You represent and warrant that you own the rights to all Content you post or that you have the right or have been given permission from the relevant copyright holder to post it under the terms of the Applicable License.</p>
-<p>You may have the opportunity to post feedback on the Service in the Meta section, including reports of erroneous behavior or requests for changes to how the Service works. If your feedback is acted upon, you will not be entitled to compensation in any form. There will be no obligation to act upon your feedback.</p>
-<p>Your use of the Service does not give you any rights in the design, branding, logos, or other design elements of the Service, and you may not copy, distribute, or otherwise use or disseminate any such elements without written permission (which may be requested by emailing info@codidact.com).</p>
-<h2 id="4-termination">4. Termination</h2>
-<p>You agree that if you engage in any conduct that contravenes these Terms, your right to use the Service may be revoked without warning. If you have provided contact details (for example, by creating an account), you may be contacted to advise you of such revocation, but there shall be no obligation to do so.</p>
-<p>Any clause that should by its nature survive termination shall be deemed to survive it, including particularly clauses 3, 4, 5, and 6.</p>
-<h2 id="5-limitations-on-liability">5. Limitations on Liability</h2>
-<p>Although every effort is made to make the Service available, the Service is provided &quot;as-is&quot;, and no guarantee is made that the Service will be available, secure, or error-free. To the maximum extent permitted by law, the Service comes with NO WARRANTY, WHETHER EXPRESS OR IMPLIED, INCLUDING THE IMPLIED WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE, TITLE, AND NON-INFRINGEMENT.</p>
-<h2 id="6-governing-law">6. Governing Law</h2>
-<p>These Terms are governed by the laws of the United Kingdom, and jurisdiction in disputes relating to the Service shall be limited to courts located in London, United Kingdom.</p>
+<!-- Placeholder: put the Terms of Service for your network here (or delete this help entry if you don't have TOS). -->
```

Comment:
Same story for the privacy policy also applies to the TOS.

Reply:
Yup, see previous reply.  You're right.